RELEASING.md

```diff
@@ -62,7 +62,7 @@ Before beginning, ensure that the docs at docs.projectcalico.org for the Calico
 
 1. Login to our operator project on connect.redhat.com and publish the operator image on the RH Catalog. This step needs to happen before we generate and submit the operator metadata bundle.
 
-1. Create the operator metadata bundle, using the tag version for VERSION and the version that the release replaces in PREV_VERSION. The versions are semver strings.
+1. Create the operator bundle, using the tag version for VERSION and the version that the release replaces in PREV_VERSION. The versions are semver strings.
```

Comment:
Could you include more notes on the PREV_VERSION?
Like if we already had bundles `1.13.0, 1.13.1, 1.14.0`: 
* If we were doing `1.13.2`, then PREV_VERSION would be `1.13.1`
* If we were doing `1.14.1`, then PREV_VERSION would be `1.14.0`
* If we were doing `1.15.0`, then PREV_VERSION would be `1.15.0`
* If we were doing `1.13.3` but `1.13.2` bundle had not been published, then would we use `1.13.1`?
I know these might all seem obvious.

Also why does the CHANNELS includes `release-`, is that because that is the branch we build the release from? That seems like an odd thing to include but maybe it is up to the packager what the CHANNELS are?

Reply:
Thanks @tmjd , I've pushed up some edits to improve the instructions based on your questions: 676d64ee10420eacb46d9b4da4679530c0453f42

> Also why does the CHANNELS includes release-, is that because that is the branch we build the release from?

It's just the convention we've been using. The channel name was arbitrary, and is used to distinguish between the different channels we have for each operator release branch. (Maybe simply `1.13` without the `release-` prefix would have been better.)

When an OCP user creates a subscription to the tigera-operator they need to specify the channel. New versions in that channel that would replace their existing version are eligible for upgrades.

Reply:
How does one upgrade from (release-)1.13 to (release-)1.14? Seems like we should maybe we should just use `stable`.

Reply:
They would update their tigera-operator subscription's channel to `release-v1.14`.

We're using multiple channels because really we have multiple "stable" streams of releases and the releases can come in any order.